pkg/codegen/go/gen.go

```diff
@@ -3560,7 +3565,8 @@ func goPackage(name string) string {
 	return strings.ReplaceAll(name, "-", "")
 }
 
-const utilitiesFile = `
+func UtilitiesFile(pkg *pkgContext) string {
```

Comment:
Nit: Any reason not to make this a method on pkgContext?

```suggestion
func (pkg *pkgContext) GenUtilitiesFile() string {
```